upgrade/upgrade.go

```diff
@@ -95,12 +101,15 @@ func upgrade(currentNodeID, namespace string, config *restclient.Config, lhClien
 		Callbacks: leaderelection.LeaderCallbacks{
 			OnStartedLeading: func(ctx context.Context) {
 				var err error
```

Comment:
Then we need to pass the error from this function to it's caller.

Reply:
Done